File: product-webapp/angularbase/carnivry/src/app/model/seat.ts
export class Seat {
    public row: number;
    public colm: number;
    public seatId: number;
    public seatPrice: number;

    constructor(row:number,colm:number, seatPrice:number){
        this.row=row;
        this.colm=colm;
        this.seatPrice=seatPrice;
    }
}

File: payment-service/src/main/java/com/stackroute/model/Seat.java
package com.stackroute.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Seat {


    private int row;
    private int colm;
    private int seatId;
    private BigDecimal seatPrice;
    private String status;
    private String seatCategory;

}
